src/Core/Services/CozyClouderyEnvService.cs

```diff
@@ -15,6 +15,11 @@ public class CozyClouderyEnvService: ICozyClouderyEnvService
         private const string INT_BASE_URI = "https://staging-manager.cozycloud.cc";
         private const string DEV_BASE_URI = "https://manager-dev.cozycloud.cc";
 
+
+        private const string PROD_STACK_OAUTHCALLBACK_URI = "https://oauthcallback.mycozy.cloud";
+        private const string INT_STACK_OAUTHCALLBACK_URI = "https://oauthcallback.cozy.works";
+        private const string DEV_STACK_OAUTHCALLBACK_URI = "https://oauthcallback.cozy.wtf";
+
```

Comment:
What I don't like with these urls, is that tomorrow, we should use oauthcallback.twake.app, lin-saas... 

So having the mycozy.cloud will result to make an extra http redirection at the end, but I don't see other way to do it right now to be honest.